plugins/sentry-backend/src/service/router.ts

```diff
@@ -22,7 +22,12 @@ export async function createRouter(
   rootLogger: Logger,
 ): Promise<express.Router> {
   const router = Router();
-  const sentryForwarder = new SentryApiForwarder('');
+  const SENTRY_TOKEN = process.env.SENTRY_TOKEN;
+  if (!SENTRY_TOKEN) {
+    console.error('Sentry token must be provided in env to start the API.');
+    process.exit(1);
```

Comment:
Throw an error instead

Reply:
and call out the expected env variable name in the error message